.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+# Issue
+
+> Describe your issue
+
+# Expected behavior
+
+> What were you expecting
+
+# How to reproduce
+
+> Add enough information to be able to reproduce your issue
```

Comment:
Typically, I'd love a reporter to write a test he'd expect to pass so that I can add it to the test suite then make it pass. 
That's if he has enough time/knowledge to write such thing though. 

Maybe: ` > Add a test case that showcases your issue and you'd expect to pass (see in test folder how to do so) or just give enough information so that the dev team can reproduce your issue`